stream-chat-android-ui-utils/api/stream-chat-android-ui-utils.api

```diff
@@ -0,0 +1,76 @@
+public final class io/getstream/chat/android/ui/utils/BuildConfig {
```

Comment:
The theme here seems to be empowering the user as we are either exposing a lot of previously `internal` as `public` or removing `@InternalStreamChatApi`

I don't necessarily mind that, but I'd just like confirmation that these is a conscious decision and that we are willing to take on any possible maintenance burden that comes out of it.